### PR TITLE
Community individual redesign

### PR DESCRIPTION
**Pull request by Electr1Xx**

### CODE REVIEW

File: src/views/Forum/components/CommunityIndividual/AboutCard.tsx [hunk 1]
```diff
@@ -91,110 +52,113 @@ export const AboutCard = ({ forum, isOwner }: IAboutProps) => {
...
+		if (!social || !url) return [];
 		return {
-			id: social?.id || '',
-			icon: social?.icon || '',
-			label: social?.label || '',
-			url: socials[key] || ''
+			id: social.id,
+			icon: social.icon,
+			label: social.label,
+			url: url
```
Comment (ArtSuslov) [inline src/views/Forum/components/CommunityIndividual/AboutCard.tsx:63]:
```suggestion
			url
```

File: src/views/Forum/components/CommunityIndividual/CommunityActions.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { IconButton } from 'src/components/IconButton';
+import ShareBox from 'src/components/ShareBox';
+import { AuthContext, IForum } from 'src/contexts';
+import { ErrorHandler } from 'src/helpers';
+import { useGilder } from 'src/hooks/useGilder';
+
+export const CommunityActions = ({ forum }: { forum: IForum }) => {
+	const { profile, loadUserInfo } = useContext(AuthContext);
+	const { following, unfollow } = useGilder();
+	const isFollowing = forum?.followers.find(follower => `0x${follower}` === profile?.id);
```
Comment (ArtSuslov) [inline src/views/Forum/components/CommunityIndividual/CommunityActions.tsx:15]: might be misleading: `following` sounds like a boolean and there is `isFollowing`

File: src/views/Forum/components/CommunityIndividual/CommunityActions.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			ErrorHandler.process(e);
+		}
+		setIsSubmitting(false);
+	};
+	return (
+		<Box display="flex" gap={0.75} alignItems="center" mt={{ xs: 0, lg: 0.6 }}>
+			{profile && (
+				<QuaternaryButton
+					sx={{
+						boxShadow: '0 0 1px #000000;'
```
Comment (ArtSuslov) [inline src/views/Forum/components/CommunityIndividual/CommunityActions.tsx:38]: does it work with semicolon?

Comment (Electr1Xx) [reply src/views/Forum/components/CommunityIndividual/CommunityActions.tsx:38]: yes

**Review by ArtSuslov — APPROVED** [review]

**Pull request merged**